packages/amplify_authenticator/lib/src/services/amplify_configure.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:amplify_flutter/amplify.dart';
+
+/// Amplify Configure Abstract Class
```

Comment:
What exactly are we using this for?

Reply:
Before having the Amplify configuration on top of the Authenticator, that file used to do all the initial configuration,  but I think that it is not bringing any meaningful use in the project anymore.